datasets/gov/aaa/oro_tarsa.csv

```diff
@@ -0,0 +1,46 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/aaa/oro_tarsa,,,,,,,,,,,,,
+,,,,,,prefix,spinta,,,,,https://github.com/atviriduomenys/spinta/issues/,,
+,,,,,,,manifest,,,,,https://github.com/atviriduomenys/manifest/issues/,,
+,,,,,,,vadovas,,,,,https://atviriduomenys.readthedocs.io/,,
+,,,,,,,,,,,,,,
+,,,,OroTarsa,,,,,,2,,,,
+,,,,,objekto_kodas,string,,,,2,open,,Ataskaitinio objekto kodas,
+,,,,,pavad,string,,,,2,open,,"Ataskaitinio objekto pavadinimas. Ūkio subjekto pagrindinės įmonės atveju nurodomas toks pat kaip JAR, o filialų atveju nurodomas rankiniu būdu",
+,,,,,x_koord,number,,,,2,open,,Ataskaitinio objekto X koordinatės pagal LKS-94,
+,,,,,y_koord,number,,,,2,open,,Ataskaitinio objekto Y koordinatės pagal LKS-94,
+,,,,,imones_kodas,string,,,,3,open,,Įmonės JAR kodas,
+,,,,,atsisk_data_nuo,integer,,,,2,open,,"Data, už kurią yra teikiamas dokumentas.",
+,,,,,,comment,type,,"update(type: ""date"", ref: ""Y"")",4,,spinta:213,,
+,,,,,mat_vnt,string,,,,4,open,,Teršalo matavimo vienetai (Klasifikatorius),
+,,,,,,enum,,,"""T""",,,,Tonos,
+,,,,,,,,,"""M""",,,,Kubiniai metrai,
+,,,,,kiek_per_ats_met,number,,,,3,open,,Per ataskaitinius metus į aplinkos orą patekęs teršalų kiekis,
```

Comment:
```suggestion
,,,,,kiek_per_ats_met,number,,,,4,open,,Per ataskaitinius metus į aplinkos orą patekęs teršalų kiekis,
```
Pateiktas pasiūlymas, jeigu šis `property` tiesiogiai susijęs su `property=mat_vnt`.
Rekomenduojama papildomai įsivertinti.